hazelcast/include/hazelcast/util/Preconditions.h

```diff
@@ -71,7 +100,7 @@ namespace hazelcast {
              */
             template<typename T>
             static const boost::shared_ptr<T> &isNotNull(const boost::shared_ptr<T> &argument,
-                                                     const std::string argName) {
+                                                         const std::string argName) {
```

Comment:
Should it be `cont std::string &`? 
I think you don't need additional string copy operation here, because you don't modify string passed here (only read operation is here).

Reply:
good catch. fixed it.